packages/link/Link.js

```diff
@@ -36,11 +36,10 @@ AvLink.defaultProps = {
 };
 
 AvLink.propTypes = {
-  url: PropTypes.string.isRequired,
   target: PropTypes.string,
   tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
   children: PropTypes.node,
-  href: PropTypes.string,
+  href: PropTypes.string.isRequired,
```

Comment:
This is a breaking change, no?

Reply:
I just released the `href` change a few hours ago and needs to be set to required. Required prop types are just warnings and don't break the apps from what I can tell.

Reply:
People are currently using this component like:
```
<AvLink url="my-url-here"
```

I don't see anywhere in this change where `url` gets picked up from props and renamed to `href`.

Reply:
https://github.com/Availity/availity-react/blob/master/packages/link/CHANGELOG.md

Reply:
mea culpa. didn't see BREAKING CHANGE in commit message